tasks/transform_conformance/tests/babel-plugin-transform-async-to-generator/test/fixtures/arguments/nested-block/input.js

```diff
@@ -0,0 +1,7 @@
+const ArrowFunction = async () => {
+  {
+    var arguments = arguments;
+    console.log(arguments);
+  }
+  console.log(arguments);
+};
```

Comment:
This one is not a TDZ violation, but `var arguments = arguments;` is still very weird code.